checka11y-minified.css

```diff
@@ -0,0 +1 @@
adius:.75rem;letter-spacing:initial;text-decoration:none;text-transform:initial;text-shadow:none;content:"WARNING (W0013): Anchor tags that contain a link to a PDF, Word, Excel, or PowerPoint document is present on the page and could be a potential accessibility issue."!important;color:var(--checka11y-text-warning);border:.4rem solid var(--checka11y-border-warning);background-color:var(--checka11y-bg-warning)}a[href*="https://spotify.com/"]::after,a[href*="https://youtu.be"]::after,a[href*="https://youtube.com"]::after{display:block;font-size:1rem;font-family:verdana,geneva,tahoma,sans-serif;font-weight:700;font-style:initial;padding:var(--checka11y-space-4) var(--checka11y-space-6);border-radius:.75rem;letter-spacing:initial;text-decoration:none;text-transform:initial;text-shadow:none;content:"WARNING (W0014): Anchor tags that contain a link to content site (Youtube, Spotify etc) is present on the document."!important;color:var(--checka11y-text-warning);border:.4rem solid var(--checka11y-border-warning);background-color:var(--checka11y-bg-warning)}section:empty::before,section>:not(h1,h2,h3,h4,h5,h6,img):last-child::before{display:block;font-size:1rem;font-family:verdana,geneva,tahoma,sans-serif;font-weight:700;font-style:initial;padding:var(--checka11y-space-4) var(--checka11y-space-6);border-radius:.75rem;letter-spacing:initial;text-decoration:none;text-transform:initial;text-shadow:none;content:"WARNING (W0012): A <section> should contain a heading element."!important;color:var(--checka11y-text-warning);border:.4rem solid var(--checka11y-border-warning);background-color:var(--checka11y-bg-warning)}section>:is(h1,h2,h3,h4,h5,h6)~:not(:is(h1,h2,h3,h4,h5,h6,img)):last-child::before{content:none!important}[title]::after{display:block;font-size:1rem;font-family:verdana,geneva,tahoma,sans-serif;font-weight:700;font-style:initial;padding:var(--checka11y-space-4) var(--checka11y-space-6);border-radius:.75rem;letter-spacing:initial;text-decoration:none;text-transform:initial;text-shadow:none;content:"WARNING (W0008): The title attribute has many accessibility concerns, consider using an alternative method."!important;color:var(--checka11y-text-warning);border:.4rem solid var(--checka11y-border-warning);background-color:var(--checka11y-bg-warning)}u::after{display:block;font-size:1rem;font-family:verdana,geneva,tahoma,sans-serif;font-weight:700;font-style:initial;padding:var(--checka11y-space-4) var(--checka11y-space-6);border-radius:.75rem;letter-spacing:initial;text-decoration:none;text-transform:initial;text-shadow:none;content:"WARNING (W0007): Using <u> element could be confused as a hyperlink, consider using a different element such as <em> or <b>."!important;color:var(--checka11y-text-warning);border:.4rem solid var(--checka11y-border-warning);background-color:var(--checka11y-bg-warning)}/*! Checka11y.css v2.0.0 | MIT License | github.com/jackdomleo7/Checka11y.css *//*! Checka11y.css v2.0.0 | MIT License | github.com/jackdomleo7/Checka11y.css */
```

Comment:
I think it'd be really good if we could match popular patterns and use the `.min.css` extension. I.e. `checka11y.min.css` 🙂 (same with the other stylesheets)

Reply:
agreed! :)